messages/root/messages.go

```diff
@@ -16,6 +16,7 @@ var (
 	RootLogLevel    = "Set the logging level, \"debug\", \"info\", or \"error\"."
 	RootFlagOut     = "Exports the output to the given <file_path/file_name.ext>"
 	RootFlagFormat  = "Changes the output format passing the json value to the flag"
+	RootFlagTimeout = "Defines how much time the CLI will wait before timing out from the HTTP connection"
```

Comment:
I know it might seem obvious, but maybe stg like this:
```suggestion
	RootFlagTimeout = "Defines how much time in seconds the CLI will wait before timing out from the HTTP connection"
```

Reply:
agreed